test/files/iface/Test.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE CPP #-}
+
+module Main where
+
+import X
+import GHC.Types
+import Control.Monad
+import Language.Haskell.TH
```

Comment:
How about adding a `qAddDependentFile` call here, since that's the primary feature in `hi` files we need to use?

Reply:
The `Test.h` is already listed in the `addDependentFile` section because of the `include`, but I can also add this!